external/netfx-conflicts/netfx-conflicts.depproj

```diff
@@ -9,6 +9,7 @@
     <PackageTargetFallback>netstandard1.1;netstandard1.2;netstandard1.3</PackageTargetFallback>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
     <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
+    <BinPlaceNETFXRuntime>false</BinPlaceNETFXRuntime>
```

Comment:
CI would just build the tests and not actually try to run them for netfx. We have an outerloop run of the netfx tests, so that one would have failed.